src/config.js

```diff
@@ -15,7 +15,16 @@ const SLIP0044 = {
   CENTRIFUGE: 0x80000001,
 }
 
+const SS58_ADDR_TYPE = {
```

Comment:
We probably want to cover all the chain that we already support (look at slip0044)
A good reference for prefixes is here: https://github.com/paritytech/substrate/blob/b2d9affd7135bdc08daba0ae460219740822980f/ss58-registry.json